<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileRefreshDialog</class>
 <widget class="QDialog" name="ProfileRefreshDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑个人资料</string>
  </property>
  <property name="fixedSize" stdset="0">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="avatarLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblAvatar">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #f0f0f0; border-radius: 70px; border: 1px solid #eee;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChangeAvatar">
       <property name="styleSheet">
        <string notr="true">padding: 6px 12px; background: #f5f5f5; border: 1px solid #ddd; border-radius: 4px;</string>
       </property>
       <property name="text">
        <string>更换头像</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="accountLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblAccountLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>账号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblUsername">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #666; background-color: #f9f9f9; padding: 6px 12px; border-radius: 4px; border: 1px solid #eee;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nicknameLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblNicknameLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>昵称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtNickname">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; padding: 6px 12px; border: 1px solid #ddd; border-radius: 4px;</string>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>请输入昵称（1-16个字符）</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="genderLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="lblGenderLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdoMale">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333;</string>
       </property>
       <property name="text">
        <string>男</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdoFemale">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333;</string>
       </property>
       <property name="text">
        <string>女</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdoUnknown">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333;</string>
       </property>
       <property name="text">
        <string>未知</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="phoneLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPhoneLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>手机号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPhone">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; padding: 6px 12px; border: 1px solid #ddd; border-radius: 4px;</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="placeholderText">
        <string>请输入手机号</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="emailLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblEmailLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #333; min-width: 60px; text-align: right;</string>
       </property>
       <property name="text">
        <string>邮箱：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtEmail">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; padding: 6px 12px; border: 1px solid #ddd; border-radius: 4px;</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>请输入邮箱</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; padding: 6px 24px; background: #2196F3; color: white; border: none; border-radius: 4px;</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="styleSheet">
        <string notr="true">font-size: 14px; padding: 6px 24px; background: #f5f5f5; border: 1px solid #ddd; border-radius: 4px;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
